ui/imports/shared/controls/AssetSelector.qml

```diff
@@ -135,7 +135,14 @@ Item {
                 cursorShape: Qt.PointingHandCursor
                 anchors.fill: itemContainer
                 onClicked: {
-                    root.selectedAsset = { address, name, balance, symbol, currencyBalance, fiatBalanceDisplay: "" }
+                    root.selectedAsset = {
+                        name: name,
+                        symbol: symbol,
+                        value: parseFloat(balance).toFixed(4),
+                        fiatBalanceDisplay: "",
+                        address: address,
+                        fiatBalance: currencyBalance
+                    }
```

Comment:
This needs to be fixed in `StatusAssetSelector` in StatusQ.
I thought we replaced `AssetSelector` with `StatusAssetSelector` already, but looks like it's not the case.

Would you mind sending the same PR in StatusQ for the time being?

Reply:
Also, surprising this wasn't an issue before...`value` hasn't been newly introduced.
Any reason we're calling `toFixed` with `4` here?

Reply:
Yes weird for me too ! I called toFIxed to discard the really long values cause it could be something like 0.3000010102920, and I kept it till 4 digits cause I see it done elsewhere, Eventually we only see 0.3 after removing the trailing zero's

Reply:
Gotcha!

Reply:
> This needs to be fixed in `StatusAssetSelector` in StatusQ. I thought we replaced `AssetSelector` with `StatusAssetSelector` already, but looks like it's not the case.
> 
> Would you mind sending the same PR in StatusQ for the time being?

Thinking if I should just fix in that and get rid of the AssetSelector in this PR itself? wdyt?

Reply:
Glad you brought it up.

Reply:
Sure go ahead. Just be aware that the `StatusAssetSelector` needs an asset source fn to determine token symbol icons

Reply:
Done :)